src/OnboardingSPA/components/SkipButton/index.js

```diff
@@ -7,7 +7,7 @@ import { useLocation, useNavigate } from 'react-router-dom';
 import { setFlow } from '../../utils/api/flow';
 import { store as nfdOnboardingStore } from '../../store';
 import { getSettings, setSettings } from '../../utils/api/settings';
-import { wpAdminPage, pluginDashboardPage } from '../../../constants';
```

Comment:
Can we get rid of `wpAdminPage` from constants in total as I don't see it being used anywhere and we are just doing `addQueryArgs` on it for no reason?

Reply:
I still think we should leave it as a fallback.